package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "clean": "rimraf node_modules",
-    "generate:plugin-dts": "tsx scripts/generate-plugin-dts.ts",
+    "generate:plugin-dts": "tsx scripts/generate-plugin-dts.ts && pnpm run lint:fix",
```

Comment:
You can revert this, cause `generate:plugin-dts` is used by the `preflight` command that should be used